packages/table/src/cell/cell.tsx

```diff
@@ -132,7 +165,15 @@ export class Cell extends React.Component<ICellProps, {}> {
         const content = <div className={textClasses}>{modifiedChildren}</div>;
 
         return (
```

Comment:
I've started using destructuring assignment in bp-chart for cases like this. e.g.:
```
{...{ style, tooltip, tabIndex, onKeyDown, onKeyUp }}
```